subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -208,24 +208,22 @@ if NRF_CLOUD_AGPS
 config NRF_CLOUD_AGPS_AUTO
 	bool "Automatically request A-GPS on bootup"
 
-config NRF_CLOUD_AGPS_SINGLE_CELL_ONLY
-	bool "Obtain approximate location using a single cell tower instead of GPS"
-
-config NRF_CLOUD_AGPS_REQ_CELL_BASED_LOC
-	bool "Request the cell-based location from nRF Cloud"
-	depends on NRF_CLOUD_AGPS_SINGLE_CELL_ONLY
-	default y
-	help
-		When disabled, nRF Cloud will obtain and store the approximate
-		cell-based location but will not send it to the device.
-		When enabled, nRF Cloud will also send the location to the
-		device.
-
 module = NRF_CLOUD_AGPS
 module-str = nRF Cloud A-GPS
 source "subsys/logging/Kconfig.template.log_config"
 
 endif # NRF_CLOUD_AGPS
+
+endmenu
+
+menu "nRF Cloud Cellular Positioning"
+
+config NRF_CLOUD_CELL_POS
+	bool "Enable nRF Cloud Cellular Positioning"
+	imply FPU
```

Comment:
Does the library really need "imply FPU"? If this is for printing floating points from the sample, this should probably be moved to the sample prj.conf.

Reply:
Some week ago in NCS FPU was disabled for APP and JSON parser could not give "double" value of latitude and longitude. Added this "imply" to make it work again. I think "imply" not "select" is acceptable for nrf_cloud_cell_pos Kconfig.